src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/AttendanceResourceIntTest.kt

```diff
@@ -370,21 +378,10 @@ class AttendanceResourceIntTest : IntegrationTestBase() {
             assertThat(it).contains("INVALID")
           }
       }
-
-      @Test
-      fun `should return bad request if trying to update a paid attendance`() {
-        val attendance = saveAttendance("COMP", courseSchedule, allocation, paidTransactionId = 123456)
-
-        webTestClient.upsertAttendance(courseSchedule.courseScheduleId, offenderBooking.bookingId)
-          .expectStatus().isBadRequest
-          .expectBody().jsonPath("userMessage").value<String> {
-            assertThat(it).contains("Attendance ${attendance.eventId} cannot be changed after it has already been paid")
-          }
-      }
     }
 
     @Nested
-    inner class SaveAttendance {
+    inner class CreateAttendance {
```

Comment:
Split save tests into create/update tests as they have different test data set up that can be shared easier in different classes.